types/bro-fs/index.d.ts

```diff
@@ -44,6 +44,8 @@ export function isSupported(): boolean;
 export function mkdir(path: string): Promise<DirectoryEntry>;
 export function readdir(path: string | DirectoryEntry, options?: {deep?: boolean}): Promise<FileEntry[]>;
 export function readFile(path: string | FileEntry, options: {type: 'ArrayBuffer'}): Promise<ArrayBuffer>;
+export function readFile(path: string| FileEntry, options: {type: "Blob"}): Promise<Blob>;
+export function readFile(path: string| FileEntry, options: {type: "File"}): Promise<File>;
```

Comment:
Please stick to using single quotes (`'`) instead of double quotes (`"`).

```suggestion
export function readFile(path: string | FileEntry, options: {type: 'File'}): Promise<File>;
```